data-migration/migrate-sharding-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,436 @@
+
+```sql
+CREATE TABLE `table5` (
+  `id` bigint(20) NOT NULL,
+  `sid` bigint(20) NOT NULL,
+  `pid` bigint(20) NOT NULL,
+  `comment` varchar(255) DEFAULT NULL,
+  INDEX (`id`),
+  UNIQUE KEY `sid` (`sid`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1
+```
+
+The following sections introduce the complete migration procedure.
+
+## Step1. Use Dumpling to export full data
+
+If you need to export multiple sharded tables belonging to the same upstream MySQL instance, you can directly use the `-f` parameter of Dumpling to export them in a single operation.
+
+If the sharded tables are stored across different MySQL instances, you can use Dumpling to export them respectively and place the results of both exports in the same parent directory.
+
+In the following example, both methods are used, and then the exported data is stored in the same parent directory.
+
+First, run the following command to use Dumpling to export table1 and table2 from my_db:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db1 -f 'my_db1.table[12]' -o ${data-path}/my_db1
+```
+
+In the command above:
+
+- `-u` or `--user` specifies the user name to be used. If a password is required for authentication, you can use `-p $YOUR_SECRET_PASSWORD` to pass the password to Dumpling.
+- `-p` or `--password` specifies the password to be used.
+- `-p` or `--port` specifies the port to be used.
+- `-h` or `--host` specifies the IP address of the data source.
+- `-t` or `--thread` specifies the number of threads for the export. Increasing the number of threads improves the concurrency of Dumpling and the export speed, and increases the database's memory consumption. Therefore, it is not recommended to set the number too large.
+- `-o` or `--output` specifies the export directory of the storage, which supports a local file path or a [URL of an external storage](/br/backup-and-restore-storages.md).
+- `-r` or `--row` specifies the maximum number of rows in a single file. If you use this parameter, Dumpling enables the in-table concurrency to speed up the export and reduce the memory usage.
+- `-F` specifies the maximum size of a single file. The unit is `MiB`. Inputs such as `5GiB` or `8KB` are also acceptable. It is recommended to keep the value to 256 MiB or less, if you use TiDB Lightning to load this file into a TiDB instance.
+- `-B` or `--database` specifies databases to be exported.
+- `-f` or `--filter` export tables that match the filter pattern. For the filter syntax, see [table-filter](/table-filter.md).
+
+> **Note:**
+>
+> If the size of a single exported table exceeds 10 GiB, it is strongly recommended to use the `-r` and `-F` options.
+
+Then, run the following command to use Dumpling to export `table3` and `table4` from `my_db2`:
```

Comment:
```suggestion
Then, run the following command to use Dumpling to export `table3` and `table4` from `my_db2`. Note that the path is `${data-path}/my_db2` instead of `${data-path}/my_db1`.
```